tests/waku_rln_relay/test_rln_group_manager_onchain.nim

```diff
@@ -257,6 +70,13 @@ suite "Onchain group manager":
     (await manager2.init()).isErrOr:
       raiseAssert "Expected error when contract address doesn't match"
 
+  asyncTest "should error if contract does not exist":
+    let manager = await setup()
+    manager.ethContractAddress = "0x0000000000000000000000000000000000000000"
+
+    expect(CatchableError):
+      await manager.init()
+
```

Comment:
this test already exists, i believe

Reply:
Will double check.

Reply:
I couldn't find it. 
Perhaps you're referring to "should error on initialization when loaded metadata does not match". I did a quick check on that one and the error message is not the same as the one in this test.